cpp/test/IceGrid/session/Server.cpp

```diff
@@ -19,7 +18,7 @@ class ClientPermissionsVerifierI final : public Glacier2::PermissionsVerifier
     {
         if(current.ctx.find("throw") != current.ctx.end())
         {
-            throw Test::ExtendedPermissionDeniedException("reason");
+            throw Glacier2::PermissionDeniedException("reason");
```

Comment:
For my understanding:
using a derived exception no longer works for this use-case because the intermediary (IceGrid) unmarshals and then remarshals the exception?

Reply:
Right.